neurom/check/morphtree.py

```diff
@@ -86,12 +87,26 @@ def is_flat(neurite, tol, method='tolerance'):
 
 
 def is_back_tracking(neurite):
-    ''' zigzag
+    ''' Check if a neurite process backtracks to a previous node. As back-tracking is taking place
```

Comment:
The sentence starting with "As" reads a bit strangely. Maybe "back-tracking takes place when..."? I think it is enough to say the volumes intersect.